Logger/Handler.php

```diff
@@ -49,7 +50,7 @@ public function __construct(
     /**
      * @inheritDoc
      */
-    public function write(array $record)
+    public function write(array $record): void
```

Comment:
Since we inherit, we have to match the parent class definition, which now includes a return value definition.